demo.md

```diff
@@ -0,0 +1,53 @@
+# Akri and Krustlet integration demo
+This is a demo to showcase the usage of a wasi discovery handler with container Akri's components.
+
+## Start your kubernetes cluster
+
+For this demo we are using microk8s, but feel free to use any of your choice, krustlet has documentation for most of them.
```

Comment:
nit - a link to kruslet documentation might be useful here (For other k8s distros, refer to kruslet documentations <link>)